R/rd-include-rmd.R

```diff
@@ -0,0 +1,35 @@
+
+block_include_rmd <- function(tag, block, env) {
+  stopifnot(is.character(tag), length(tag) == 1, !is.na(tag))
+  md_path <- tempfile(fileext = ".md")
+  on.exit(unlink(md_path, recursive = TRUE), add = TRUE)
+  if (!requireNamespace("rmarkdown", quietly = TRUE)) {
+    stop("@includeRmd requires the rmarkdown package")
+  }
+  rmd_path <- rmd_process_links(tag)
+  on.exit(unlink(rmd_path, recursive = TRUE), add = TRUE)
+  rmarkdown::render(rmd_path, output_format = rmarkdown::github_document(),
```

Comment:
Can you please use tidyverse style indenting here? http://style.tidyverse.org/syntax.html#indenting

Reply:
Ooops, sure.